axelor-crm/src/main/resources/views/Dashboard.xml

```diff
@@ -494,4 +549,31 @@
   		<view type="chart" name="chart.customer.crm.activities"/>  		
 	</action-view>
 	
+	<action-view name="dashlet.salesman.dashboard.planned.activities" title="My Planned Activities"
+		model="com.axelor.apps.crm.db.Event">
+		<view type="grid" name="my-event-grid"/>
+		<view type="form" name="event-form"/>
+	    <domain>self.statusSelect = 1 and self.user =:__user__</domain>
+	</action-view>
+	
+	<action-view name="dashlet.salesman.dashboard.activities.events.by.team" title="Events by team"
+		model="com.axelor.apps.crm.db.Event">
+		<view type="grid" name="my-event-grouped-by-team-grid"/>
```

Comment:
Need to add hidden team field in grid to display correctly.
also check for orderBy.